README.md

```diff
@@ -55,3 +55,72 @@ See `k8s-jenkins-cac.tf` in your project.
 There are 2 environments involved:
 * *target*: this is where the services will be deployed (where the version changes happen).
 * *upstream*: this is where the system tests are run. It is also the source for the *versions* of the apps.
+
+---
+
+#### Environment Config
+
+An "environment" is the set of machines where your microservices will run. For example: you will probably have some environments: "development", "staging" and "production". In a cloud provider like GCP, we recommend separating the environments on a project level. This means that one gcp project correlates to one kuberpult environment 1:1 - although this is not a technical requirement.
+
+Environments are also called "stages" - but in kuberpult we stick to "environments", short "envs".
+
+The config for an environment is stored in a json file called `config.json`. This file would go into an environment in the manifests repository like this: `environments/development/config.json` (in this example the `config.json` file would dictate the configuration for the `development` environment)
+
+In the `config.json` file there are 3 main fields:
+- [Upstream](#upstream)  `"upstream"`
+- [ArgoCD](#argocd)    `"argocd"`
+- [EnvironmentGroup](#environment-group) `"environmentGroup"`
+
+##### Upstream:
+
+The `"upstream"` field can have one of the two options (cannot have both):
+  - `latest`: can only be set to `true`
+  - `environment`: has a string which is the name of another environment. Following the chain of upstream environments would take you to the one with `"latest": true`
```

Comment:
explain that this is used in the release train

Reply:
Done :)
Thanks!